ci/git-problem-matcher.json

```diff
@@ -0,0 +1,16 @@
+{
+    "problemMatcher": [
+        {
+            "owner": "git-test-suite",
+            "pattern": [
+                {
+                    "regexp": "^([^ :]+\\.sh):(\\d+): (error|warning|info):\\s+(.*)$",
+		    "file": 1,
```

Comment:
mixed tab and space, I fixed it when cherry-picking.

Reply:
Thank you!